### DISCUSSION TITLE
Unable to upload a proof for github student

### DISCUSSION THREAD

**Discussion by CyberMatmat** [community/community#108011] [GitHub Education]
### Hi everyone,

Hi everyone,

I'm a current IT student facing an issue while renewing my GitHub Student Developer Pack. I successfully completed my studies last year and received the pack then. Now, during renewal, I'm encountering a problem with submitting my application.

Whenever I upload my proof of enrollment (tried with my school inscription certificate, contract, and transcript of records) and click "Process Application," the page refreshes and I'm taken back to the initial upload step. It seems my application isn't being submitted at all.

I'm using Google Chrome on Windows 10 and have tried clearing my browser cache and cookies, but the issue persists. I've also searched the GitHub support documentation and community forums for similar problems but haven't found any solutions.

Any guidance or suggestions on how to resolve this issue would be greatly appreciated.

Thanks in advance,

Mattéo

**Comment by amankr1469** [comment]
Had this same issue. If you look at the browser console you may see an error there. I had an issue trying to submit my transcript and was getting a 413 Payload too Large POST error. You should try to resize your screenshots if you submitted any.
